frontend/packages/webterminal-plugin/integration-tests/features/web-terminal/web-teminal-basic.feature

```diff
@@ -1,4 +1,4 @@
-@web-terminal
+@web-terminal @dev-console
```

Comment:
This is not part of devconsole package anymore

Reply:
Sure, removed the tags!